2.0/0x200-Components/0x203-C3-Oracle.md

```diff
@@ -25,4 +25,6 @@ Category “C3” lists requirements related to the Oracle smart contract as one
 
 For more information, see also:
 
-* [The Dangers of Price Oracles in Smart Contracts](https://www.youtube.com/watch?v=YGO7nzpXCeA)
\ No newline at end of file
+* [ORACLES](https://ethereum.org/en/developers/docs/oracles/)
+* [The Dangers of Price Oracles in Smart Contracts](https://www.youtube.com/watch?v=YGO7nzpXCeA)
+* [So you want to use a price oracle](https://samczsun.com/so-you-want-to-use-a-price-oracle/)
```

Comment:
This should go to Interactions (with oracles) I guess.

Reply:
I assume that to design a secure oracle, we need to be aware of the risks of integration. I am happy to add a guide on building secure oracle as soon as I find one :D